Sources/Common/Logging.swift

```diff
@@ -36,6 +36,7 @@ extension OSLog {
     }()
 
     public enum Categories: String, CaseIterable {
+        case contentBlocking = "Content Blocking"
```

Comment:
Using the same name as in macOS app repo will shows "Ambiguous use of 'contentBlocking" when I drop BSK into macOS project.